### PR TITLE
test: fix code coverage report

### PR DESCRIPTION
**Pull request by barmac**
### Proposed Changes

The code coverage is currently not collected at all. This PR fixes the problem.

### Checklist

To ensure you provided everything we need to look at your PR:

* [x] **Brief textual description** of the changes present
* [ ] **Visual demo** attached
* [ ] **Steps to try out** present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [ ] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

File: karma.config.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ module.exports = function(karma) {
                 ].concat(coverage ? [
                   [ 'istanbul', {
                     include: [
-                      'lib/**'
+                      'src/**'
```
Comment (philippfromme) [inline karma.config.js:84]: Shouldn't this config and https://github.com/bpmn-io/dmn-js-properties-panel/blob/main/karma.config.js be the same ideally?

Comment (barmac) [reply karma.config.js:84]: Indeed, we could port this to the dmn project too.

Comment (barmac) [reply karma.config.js:84]: Oh, in DMN it just works. I guess there's no need to fix it then.

Comment (barmac) [reply karma.config.js]: Now this project will mirror the dmn-js-properties-panel settings. Thanks for the suggestion.

**Pull request merged**